Xamarin.Forms.Core/IDecorableText.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Xamarin.Forms
+{
+    public interface IDecorableText
```

Comment:
Spaces => tabs

Reply:
use `IDecorableTextElement` for consistency